scripts/module/keybinder.nut

```diff
@@ -39,26 +43,26 @@ function partition( input, separator ) {
 function onPlayerCommand( player, cmd, text ) {
   cmd = cmd.tolower();
   if (cmd == "bind") {
-    text = text.tolower();
-    local bindTable = {};
-    if (text.len() == 0) {
+    if (!text) {
       local currbinds = "Current binds: \n";
       if (binds[player.ID]) {
         foreach (key, bind in binds[player.ID]) {
-          currbinds += key + ": " + bind + "\n";
+          currbinds += "'" + key + "'" + ": '" + bind + "'\n";
         }
       }
       MessagePlayer( currbinds + "Usage:\n" +
```

Comment:
Why this space between `(` and `currbinds` ?

Reply:
Misclick and lack of proofreading on my part. Fixed!